website/docs/installing-podman-desktop/index.md

```diff
@@ -0,0 +1,406 @@
+
+  ```powershell
+  > wsl --install --no-distribution
+  ```
+
+#### Procedure
+
+1. [Download the Windows installer for restricted environments for your platform](/downloads/windows).
+2. Copy the downloaded file to the Windows host in a restricted environment.
+3. Run the downloaded file to install Podman Desktop.
+4. Open Podman Desktop from the desktop icon or the **Windows** menu.
+5. Click the **Install** button to install Podman.
+
+#### Verification
+
+- Open Podman Desktop from the desktop icon.
+- Open Podman Desktop from the **Windows** menu.
+
+#### Next steps
+
+- [Configure Podman Desktop to run behind your proxy](/docs/proxy/using-a-proxy).
+- [Configure Podman to run behind your VPN](/docs/proxy/using-a-vpn-on-windows).
+
+</TabItem>
+
+<TabItem value="chocolatey" label="Chocolatey">
+
+#### Prerequisites
+
+- Windows 10 or 11.
+- [Chocolatey package manager](https://chocolatey.org/install).
+
+#### Procedure
+
+- Run the command from the command line to install [Podman Desktop Chocolatey package](https://community.chocolatey.org/packages/podman-desktop):
+
+  ```shell-session
+  $ choco install podman-desktop
+  ```
+
+#### Verification
+
+- Open Podman Desktop from the desktop icon.
+- Open Podman Desktop from the **Windows** menu.
+
+</TabItem>
+
+<TabItem value="winget" label="Winget">
+
+#### Prerequisites
+
+- Windows 10 or 11.
+- [Winget Package manager for Windows](https://aka.ms/getwinget).
+
+#### Procedure
+
+- Run the command from the command line to install [Podman Desktop Winget package](https://winget.run/pkg/RedHat/Podman-Desktop):
+
+  ```shell-session
+  $ winget install -e --id RedHat.Podman-Desktop
+  ```
+
+#### Verification
+
+- Open Podman Desktop from the desktop icon.
+- Open Podman Desktop from the **Windows** menu.
+
+</TabItem>
+
+<TabItem value="windows-scoop" label="Scoop">
+
+#### Prerequisites
+
+- Windows 10 or 11.
+- [Scoop package manager](https://github.com/ScoopInstaller/Install#readme).
+
+#### Procedure
+
+- Run the commands from the command line to install [Podman Desktop Scoop package](https://scoop.sh/#/apps?q=podman-desktop&s=0&d=1&o=true)
+  :
+
+  ```shell-session
+  $ scoop bucket add extras
+  $ scoop install podman-desktop
+  ```
+
+#### Verification
+
+- Open Podman Desktop from the desktop icon.
+- Open Podman Desktop from the **Windows** menu.
+
+</TabItem>
+
+</Tabs>
+
+</TabItem>
+
+<TabItem value="mac" label="macOS">
+
+<Tabs groupId="alternatives">
+
+<TabItem value="universal-dmg" label="Universal Disk Image">
+
+#### Prerequisites
+
+- macOS.
+
+#### Procedure
+
+1. [Download the Universal `.dmg` file](/downloads/macos).
+1. Open the downloaded file to open the disk image.
+1. Drag the Podman Desktop icon to the Applications folder.
+
+#### Verification
+
+- Open Podman Desktop from **Launchpad**.
+- Open Podman Desktop from the **Applications** directory.
+
+</TabItem>
+
+<TabItem value="intel-dmg" label="Intel Disk Image">
```

Comment:
Intel Disk Image? Seems a bit weird? Never heard of a .dmg be described like that.

Can we just say "Apple" and then "Apple (Silicon)" which is usually the standard?